src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -117,7 +119,10 @@ private void initialize() {
      */
     @FXML
     private void handleDeletePersons() {
-        final List<ReadOnlyViewablePerson> selected = personListView.getSelectionModel().getSelectedItems();
+        final List<ReadOnlyViewablePerson> selected = new ArrayList<>(personListView.getSelectionModel()
+                                                                                    .getSelectedItems());
+        selected.removeIf(p -> !(p instanceof  ReadOnlyViewablePerson));
```

Comment:
Perhaps remove this line and change the below if-check:
`if (selected.isEmpty() || selected.contains(null))`